### PR TITLE
amazon bedrock: generate api docs

### PR DESCRIPTION
**Pull request by dfokina**
Fixes https://github.com/deepset-ai/haystack-core-integrations/issues/302

### CODE REVIEW

File: integrations/amazon_bedrock/pydoc/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+loaders:
+  - type: haystack_pydoc_tools.loaders.CustomPythonLoader
+    search_path: [../src]
+    modules: [
+      "haystack_integrations.components.generators.amazon_bedrock.generator",
```
Comment (ZanSara) [inline integrations/amazon_bedrock/pydoc/config.yml:5]: I think we should document all files in the repo:

```suggestion
      "haystack_integrations.components.generators.amazon_bedrock.generator",
      "haystack_integrations.components.generators.amazon_bedrock.adapter",
      "haystack_integrations.components.generators.amazon_bedrock.errors",
      "haystack_integrations.components.generators.amazon_bedrock.handler",
```

**Review by ZanSara — CHANGES_REQUESTED** [review]
I think some modules are missing from the config

File: integrations/amazon_bedrock/pydoc/config.yml (hunk 1 above)
Comment (dfokina) [reply integrations/amazon_bedrock/pydoc/config.yml:5]: @ZanSara added! (I added the import paths with the file names, e.g. adapter**s** and handler**s** (multiple), is that right 🤔 ?)

Comment (ZanSara) [reply integrations/amazon_bedrock/pydoc/config.yml:5]: Oh that's right!

**Review by ZanSara — APPROVED** [review]
LGTM

**Pull request merged**